nodewatcher/modules/importer/nw2/management/commands/import_nw2.py

```diff
@@ -277,7 +277,7 @@ def import_pools(self, data):
             pool_mdl = pool_models.IpPool(
                 family='ipv4',
                 network=pool['network'],
-                prefix_length=pool['cidr'],
+                prefix_length=pool['ip_space'],
```

Comment:
That should not be touched from what I understand.
That is only used to import old Nodewatcher 2 JSON dump for migration